cogs/silly.py

```diff
@@ -1,10 +1,29 @@
 import random
 import aiohttp
 import discord
+from math import *
+import re
+import operator
 from discord.ext import commands
 
 from .utils import auth, AuthPerms, AuthType
 
+safe_dict = dict()
```

Comment:
```suggestion
safe_dict = {}
```

(This is faster, believe it or not.)